### PR TITLE
Closes #34 Add DDEV support and add DDEV instructions to documentation.

### LINKED ISSUE #34: Add DDEV instructions to contributing documentation
**Do not disclose security vulnerabilities here, please email us instead: 
az-digital-security@list.arizona.edu**

## Is your feature request related to a problem? Please describe.
Currently, there are only local development instructions for Lando; however, there's a known issue for Lando and Pantheon, which doesn't allow a user to access information for a given Pantheon site, so providing an alternative local development process is necessary for Pantheon users who might also want to contribute to Quickstart.

## Describe the solution you'd like
Add basic instructions for creating a local working copy of a Quickstart site using DDEV.

## Describe alternatives you've considered
There are already instructions for Lando, but as listed above, Lando has some limitations when used with Pantheon sites and some users prefer DDEV to Lando, so offering an alternative is useful.

## Additional context
N/A

### PR DESCRIPTION
**Pull request by joshuasosa**
## Description
Adds local development config for DDEV support along the lines of Lando support. Adds DDEV instructions to contributing documentation.

## Related Issue

Issue #34 
## How Has This Been Tested?

Tested locally on Windows by @joshuasosa 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .ddev/commands/web/install [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+## Description: Run the az_quickstart installation inside the web container.
+## Usage: install
+## Example: "ddev install"
+
+rm -rf /var/www/html/docroot && rm /var/www/html/index.html
+git clone https://github.com/az-digital/az-quickstart-scaffolding.git /var/www/html
```
Comment (joeparsons) [inline .ddev/commands/web/install]: Do we want to add the step that checks if a branch of the same name exists for the scaffolding repo here (that we use in or Lando and ProboCI configs)?

See: https://github.com/az-digital/az_quickstart/blob/master/.lando.yml#L22 and https://github.com/az-digital/az_quickstart/blob/master/.probo.yaml#L10

**Comment by joshuasosa** [comment]
> FYI I updated ddev from 1.11.2 -> 1.13.0 and it now works.

Tested on Mac with 1.13.0 and confirmed it also works there. I think I had to do `brew install mkcert nss` before it wanted to go through. But `ddev install` worked fine for me after that. (I also did `lando poweroff` before doing anything with ddev.)

**Comment by tadean** [comment]
I tested this out on Ubuntu 18.04 LTS. Initially I ran into the following issue when trying to use `ddev install`:

```
/mnt/ddev_config/commands/web/install: Permission denied
Failed to run install : exit status 126
```
I think this is because the git repo doesn't currently have the execute bit committed on `.ddev/commands/web/install` and `.ddev/commands/web/drush`

I'd perhaps suggest setting the execute bit on those files. This might not be reproducible on OSX/Windows due to the network share to the container making permissions a bit different when the container runs the script there.

Aside from that minor issue, this seemed to work great after setting those permissions! The install process ran successfully and the composer package ended up as a local container symlink, with a local composer repository set up. Good job @joshuasosa !

File: .ddev/commands/web/install (hunk 1 above)
Comment (joshuasosa) [reply .ddev/commands/web/install]: I added in the same line from the lando file. I also added in the other tooling that's in that file, namely phpcs, drupal, and drupal-check.

Comment (joeparsons) [reply .ddev/commands/web/install]: Looks good.  Thanks!

**Comment by joshuasosa** [comment]
> I tested this out on Ubuntu 18.04 LTS.
I think I have all the needed changes in, including making those command files executable. I was able to successfully install quickstart on Windows and Mac. If you're able to test again on Ubuntu/Linux, it'd be much appreciated!

**Comment by tadean** [comment]
This worked successfully for me on Linux after the execute bit permission changes! I also checked out the new `drupal-check`, `phpcs` and `drupal` console commands and they all seemed to work.

Good work on this @joshuasosa !

**Pull request merged**